webui/react/src/utils/routes.ts

```diff
@@ -0,0 +1,11 @@
+export const isFullPath = (url: string): boolean => {
+  return url.startsWith('http');
+};
+
+export const parseUrl = (url: string): URL => {
+  if (!isFullPath(url)) {
+    if (!url.startsWith('/')) url = '/' + url; // TODO assume url is absolute, or we could throw
```

Comment:
suggestion: could be a useful function
```
export const isAbsolutePath = (url: string): boolean => url.startsWith('/');
```

Reply:
thoughts about handling relative urls? assume they're absolute or throw